dlatk/dlaConstants.py

```diff
@@ -274,6 +274,16 @@ def fit_transform(self, X, y=None):
     lambda d: 2*sqrt(d+3/float(8))
 ]
 
+## PyMallet defaults
+DEF_STOPLIST_LANGUAGE_CHOICES = ['en', 'de', 'fi', 'fr', 'jp', 'nl']
+DEF_STOPLIST_LANGUAGE = 'en'
+DEF_NUM_TOPICS = 100
+DEF_NUM_ITERATIONS = 1000
+DEF_ALPHA = 0.05
+DEF_BETA = 0.01
+DEF_OUTPUT_STATE_FILE = 'lda_model.state'
```

Comment:
Since we try to never have data in home directories, it's probably best to default to /tmp/lda_model.state ...etc...